lib/src/presentation/bloc/card_bloc.dart

```diff
@@ -15,7 +17,6 @@ class CardBloc extends ICardBloc {
   int currentPage = kCardBlocStartingCurrentPage;
   late List<HearthstoneCard> allCards = [];
   int numberOfPagesCardList = kDefaultNumberOfPagesCardList;
-
   Map<int, List<HearthstoneCard>> page = {};
 
   CardBloc({required this.repositoryCard});
```

Comment:
implement a usecase call instead of calling straight to the repository, to keep the correct Clean implementation